### DISCUSSION TITLE
Raw Bootstrap and no Blazorise Controls working

### DISCUSSION THREAD

**Discussion by Domeman** [Megabit/Blazorise#5559] [General]
I tried to get Blazorise to work with Blazor server Authentication. I fixed all the issues but I don't get the default Blazorise Home page.
I am new to Blazorise and Blazor, nay help would be appreciated. Thanks.

My page source looks like this . . . 

<style id='b-theme-variables'>
  | --b-theme-white: #ffffff;
  | --b-theme-black: #343a40;
  | --b-border-radius: .25rem;
  | --b-border-radius-lg: .3rem;
  | --b-border-radius-sm: .2rem;
  | --b-theme-primary: #0288D1;
  | --b-theme-primary-r: 2;
  | --b-theme-primary-g: 136;
  | --b-theme-primary-b: 209;
  | --b-theme-primary-a: 1.00;
  | --b-button-primary-background: #0288D1;
  | --b-button-primary-border: #0288D1;
  | --b-button-primary-hover-background: #0173B1;
  | --b-button-primary-hover-border: #349FDA;
  | --b-button-primary-active-background: #016CA7;

**Comment by stsrki** [comment]
You might need to enable Interactive Server mode

See https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#render-modes

**Reply by David-Moreira** [reply]
To add on to this, it's best if you start from a Blazorise working point and then add things on top.
So I'd recommend getting started with one of our ready to go templates. We actually have authentication as something we want to add to our templates but haven't gotten around to it. Anyway hopefully you can copy the stuff on top of a working template, I think that'll be easier.
https://blazorise.com/docs/templates

**Reply by Domeman** [reply]
I need authentication for my app. 

I did take your template and tried to make it work with Indenity. 
![image](https://github.com/Megabit/Blazorise/assets/441108/cb2c8bd6-3658-42a2-a399-468f1d6d0bfb)

It appears that the scripts are not loading. 

Thanks.

**Reply by David-Moreira** [reply]
Please go one step at a time.
Do you mean to say, that you are using a **Blazorise template** **without any custom changes** whatsoever and it's not loading the scripts?
- Can you let us know the template options that you've chosen.
- Can you check your network tab and see if the javascript resources are being blocked by something on your end?
